secure/net/src/lib.rs

```diff
@@ -25,9 +25,9 @@ use thiserror::Error;
 
 /// Read, Write, Connect timeout in seconds.
 #[cfg(not(test))]
-const TIMEOUT: u64 = 30;
+pub const TIMEOUT: u64 = 30;
 #[cfg(test)]
-const TIMEOUT: u64 = 5;
+pub const TIMEOUT: u64 = 5;
```

Comment:
actually, now with this, we should probably move this value into the the test module below

Reply:
Simply removing it.